includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -3735,7 +3760,7 @@ private function is_refund_request() {
 	 * @return string The redirect URL.
 	 */
 	protected function get_redirect_url( $return_url, $payment_intent, $payment_information, $order, $payment_needed ) {
-		$selected_payment_type = $this->oc_enabled ? $payment_information['payment_method_details']->type : $payment_information['selected_payment_type'];
+		$selected_payment_type = $this->is_oc_active() ? $payment_information['payment_method_details']->type : $payment_information['selected_payment_type'];
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $selected_payment_type. Keep variable name length under 20. (undefined)

(LongVariable)